dist/grafana/grafana-prometheus-zrepl.json

```diff
@@ -1292,171 +1367,141 @@
+        "y": 42
       },
-      "hiddenSeries": false,
       "id": 19,
-      "legend": {
-        "avg": false,
-        "current": false,
-        "max": false,
-        "min": false,
-        "show": true,
-        "total": false,
-        "values": false
-      },
-      "lines": true,
-      "linewidth": 1,
-      "links": [],
-      "nullPointMode": "null",
       "options": {
-        "alertThreshold": true
+        "alertThreshold": true,
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
       },
-      "percentage": false,
-      "pluginVersion": "9.3.6",
-      "pointradius": 5,
-      "points": false,
-      "renderer": "flot",
-      "seriesOverrides": [],
-      "spaceLength": 10,
-      "stack": false,
-      "steppedLine": false,
+      "pluginVersion": "11.3.0",
       "targets": [
         {
           "datasource": {
             "type": "prometheus",
-            "uid": "${DS_PROMETHEUS}"
+            "uid": "${DS_MIMIR}"
           },
           "expr": "go_goroutines{job='$prom_job_name'}",
           "format": "time_series",
           "intervalFactor": 1,
           "refId": "A"
         }
       ],
-      "thresholds": [],
-      "timeRegions": [],
       "title": "number of goroutines (should not grow unboundedly)",
-      "tooltip": {
-        "shared": true,
-        "sort": 0,
-        "value_type": "individual"
-      },
-      "type": "graph",
-      "xaxis": {
-        "mode": "time",
-        "show": true,
-        "values": []
-      },
-      "yaxes": [
-        {
-          "format": "short",
-          "logBase": 1,
-          "min": "0",
-          "show": true
-        },
-        {
-          "format": "short",
-          "logBase": 1,
-          "show": true
-        }
-      ],
-      "yaxis": {
-        "align": false
-      }
+      "type": "timeseries"
     }
   ],
-    "refresh": "5s",
-  "schemaVersion": 37,
-  "style": "dark",
+  "refresh": "5s",
+  "schemaVersion": 40,
   "tags": [],
   "templating": {
     "list": [
       {
         "current": {},
         "datasource": {
           "type": "prometheus",
-          "uid": "${DS_PROMETHEUS}"
+          "uid": "${DS_MIMIR}"
         },
-        "definition": "",
-        "hide": 0,
-        "includeAll": false,
+        "definition": "label_values(zrepl_replication_last_successful,job)",
+        "includeAll": true,
         "label": "Prometheus Job Name",
-        "multi": false,
+        "multi": true,
         "name": "prom_job_name",
         "options": [],
         "query": {
-          "query": "label_values(up, job)",
-          "refId": "Prometheus-prom_job_name-Variable-Query"
+          "qryType": 1,
+          "query": "label_values(zrepl_replication_last_successful,job)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
```

Comment:
> Fix the prometheus job name can be anything because we use up instead of a zrepl specific metric

That is happening here.

The problem is that this doesn't work for recv-side only processes.

We need to use a metric that is exposed by all zrepl daemons.

Reply:
`zrepl_start_time` is such a metric